sdk/keyvault/keyvault-certificates/src/index.ts

```diff
@@ -197,7 +194,7 @@ export {
   ListPropertiesOfIssuersOptions,
   ListDeletedCertificatesOptions,
   MergeCertificateOptions,
-  PipelineOptions,
+  CommonClientOptions,
```

Comment:
It's interesting that we export PipelineOptions - wouldn't replacing that with CommonClientOptions be a breaking change?

Reply:
I think it will, but PipelineOptions is not longer on coreV2
However, now that I think of it, I can change `CommonClientOptions` here and replace it with `ExtendedCommonClientOptions` from Core-http-compat.
CC: @timovv